src/zenml/constants.py

```diff
@@ -119,6 +121,9 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 SEGMENT_KEY_DEV = "mDBYI0m7GcCj59EZ4f9d016L1T3rh8J5"
 SEGMENT_KEY_PROD = "sezE77zEoxHPFDXuyFfILx6fBnJFZ4p7"
 
+# ZenML Analytics Server - URL
+ANALYTICS_SERVER_URL = "https://zenml-analytics-server-76jzsesopq-lm.a.run.app"
```

Comment:
I just ran some tests by switching this URL (simulating our server is not reachable) and it crashes every command I run. I'm not sure where best to implement this, but we need some error handling there and failure to send analytics or dump to json should never lead to a failure of the product or even be visible to the user.

Reply:
First of all, this is definitely not the hostname we should use, because it is not guaranteed to be the same if we re-deploy the analytics server with Cloud Run in the future. We need to map a DNS alias to whatever the Cloud Run server ends up using.